### ISSUE TITLE
Auto Folder Collapse does not work with Obsidian v1.7.4

### ISSUE THREAD

**Issue by hubuque** [DarioCasciato/obsidian-auto-folder-collapse#2]
更新V1.7.4后插件失效了

**Comment by DarioCasciato** [comment]
Hello @hubuque, I was able to recreate the error case. Thank you for opening the issue.
The problem will be examined and resolved this week (best case).

Regards
Dario

**Comment by Scoubines** [comment]
Similar to the comment, it's not working on my side too and makes the app starting time much longer (so I disabled it for now)

Thank you so much for your time!

**Comment by DarioCasciato** [comment]
@Scoubines @hubuque 

The Bugfix was merged. New tag is going to be created in the next hour. 

So this Issue is going to be closed.

Regards Dario

**Comment by DarioCasciato** [comment]
New Version Tag is "1.1.1".

**Comment by Scoubines** [comment]
I'm not sure I catch right the tag thing, does it mean we'll able to make the 1.1.1 update in 1 hour?

**Comment by DarioCasciato** [comment]
> I'm not sure I catch right the tag thing, does it mean we'll able to make the 1.1.1 update in 1 hour?

You can already make the update!
1. Open the "community plugins" settings in obsidian
2. There is a "check for updates" buttom. Click it
3. The auto folder collapse plugin gets a "update" button in the plugins list. Click it

After this, the plugin should work normally again and not cause any delays!

Regards
Dario

**Comment by Scoubines** [comment]
Oh thank you!
But I can't see the new update on my side, is it only me?
I just took a screenshot to show you. I tried uninstalling the app and check again, same. Any ideas?
![Update](https://github.com/user-attachments/assets/415faefa-8757-4593-8c1a-158e1bad6935)

**Comment by DarioCasciato** [comment]
> Oh thank you!
> But I can't see the new update on my side, is it only me?
> I just took a screenshot to show you. I tried uninstalling the app and check again, same. Any ideas?
> ![Update](https://github.com/user-attachments/assets/415faefa-8757-4593-8c1a-158e1bad6935)
> 

Hmm it looks like that i made an error in the manifest.json file... I forgot to change the version to 1.1.1😂

But does the plugin work? I'll update the version number as soon as i get home.

Regards
Dario

**Comment by Scoubines** [comment]
Ah okay, no worries, take your time!

Even with the last wrongly named 1.1.0 it doesn't work for me

Best,

**Comment by DarioCasciato** [comment]
Hi @Scoubines , if you check for updates now, there is the version1.1.2 of the plugin.
I tested the new plugin version with obsidian v1.7.4 on another machine and it seems to be working fine.
If the problem still persists, try to execute "Reload app without saving" command, when you open the command pallette with "ctrl+p".
If there are still any issues, please comment here, since i reopened the issue👍

Regards
Dario

**Comment by Scoubines** [comment]
So I've tried with personal vault and it didn't work with 1.1.2 but strangely enough it works with another random vault. So I guess it must be come from a conflict with either one of my snippets and community plugin? Is there anything I sent it here like a log to have you look at the issue?
It used to work fine before btw

Thank you again for your support!

**Comment by DarioCasciato** [comment]
@Scoubines hmm you it also sounds for me like a conflict🤔

There could ve something in the console popping up. Open the vault that is causing the problem, and open the developer console with the shortcut "shift+ctrl+i".
From there, reload obsidian with the reload command and then open some folders and close the parent folder.
Best case, there are errors getting printed in the developer console. If there are error popping up, you could send them in here.

Im glad that i can help, and thank you for pointing out!

Regards
Dario

**Comment by Scoubines** [comment]
Hi!

Thank you for the follow-up! So I have done the procedure and joined the log file. Also here the passage where it loads Auto folder which gets a few errors after as you see.
![screen](https://github.com/user-attachments/assets/cbfb2eb6-fdfc-4a21-9150-07eed809afeb)

Thank you for your insights!

[obsidian.md-1729200628611.log](https://github.com/user-attachments/files/17423153/obsidian.md-1729200628611.log)

**Comment by DarioCasciato** [comment]
Hi @Scoubines, i tried to find/recreate the problem for quite some time, but i couldnt find the driver for this problem. could you send me a list of all the plugins that you are using? In the meantime i'll try some other tests.

thanks in advance and Regards
Dario

**Comment by DarioCasciato** [comment]
Hi @Scoubines, in the meantime i updated the plugin to version 1.1.4 and fixed some things. please update the plugin and check if the problem still persists.

Regards
Dario

**Comment by Scoubines** [comment]
Hi @DarioCasciato,

Thank you for taking the time to look into it! So before sending you the list, I've tried myself to disable (by moving plugins and Snippets to another folder) and putting them back 1 by 1 and surprise it's working again now...really an odd bug that I had. So I've made a few relaunch of the app to make sure it stays like that and so far so good!

I might get back to you if the bug is back some day tho,
Grazie mille!
